runtime/interpreter/interpreter.go

```diff
@@ -1963,18 +1963,16 @@ func (interpreter *Interpreter) declareTypeRequirement(
 	// Evaluate nested declarations in a new scope, so values
 	// of nested declarations won't be visible after the containing declaration
 
-	(func() {
```

Comment:
I guess this refactor is OK, but I'm not 100% sure: Are we sure that the part after the initial end of this nested function (https://github.com/onflow/cadence/pull/948/files#diff-a36f730b18ca31316909bd74125003c1fbe785a13f926b4a0631e7c0cc6a51e4L1977) will not be affected by the nested declarations being in scope?

Reply:
yeah, probably we can leave it as it is then. This change is anyway not related to the identifier change.